gcd/src/main/java/org/spine3/server/storage/datastore/DatastoreStorageFactory.java

```diff
@@ -95,28 +133,24 @@ public void close() throws Exception {
         // NOP
     }
 
-    public Options getOptions() {
-        return options;
+    protected DatastoreWrapper getDatastore() {
+        return datastore;
     }
 
-    @SuppressWarnings("WeakerAccess") // We provide it as API
-    public static class Options {
-        private Options() {}
-
-        private static final int DEFAULT_PAGE_SIZE = 10;
-
-        private int pageSize = DEFAULT_PAGE_SIZE;
-
-        public void setEventIteratorPageSize(int pageSize) {
-            if (pageSize < 1) {
-                throw new IllegalArgumentException("Events page can not contain less than one event.");
-            }
+    protected void setDatastore(DatastoreWrapper datastore) {
+        this.datastore = datastore;
+    }
 
-            this.pageSize = pageSize;
-        }
+    private static class StandStorageRecord extends Entity<String, EntityStorageRecord> {
```

Comment:
Please add class-level Javadocs. It is not totally clear why you would need such a class and why it is hidden from any user customisation.